python/tskit/vcf.py

```diff
@@ -111,6 +112,20 @@ def __init__(
                 sample_mask = np.array(sample_mask, dtype=bool)
                 self.sample_mask = lambda _: sample_mask
 
+        # The VCF spec does not allow for positions to be 0, so we error if one of the
+        # transformed positions is 0 and allow_position_zero is False.
```

Comment:
Sufficient to check first site, I think?

Reply:
I think we need to check them all, as the transform function could have shuffled the order.